src/App.tsx

```diff
@@ -1,61 +1,104 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { ChangeEvent, FormEvent, useState } from 'react';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [text, setText] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [newTodos, setNewTodos] = useState(todos);
+
+  const handleUserChange = event => {
```

Comment:
The `handleUserChange` function should specify the type of the `event` parameter, such as `ChangeEvent<HTMLSelectElement>`, to ensure type safety and clarity.